Tests/Unit/Controller/SitemapControllerTest.php

```diff
@@ -86,13 +86,15 @@ public function testRequestJson()
 
     public function testRequestXml()
     {
-        $response = new Response('some-xml-string');
-        $this->templating->expects($this->once())->method('render')->will($this->returnValue($response));
+        $this->templating->expects($this->once())
+            ->method('render')
+            ->with($this->equalTo('CmfSeoBundle:Sitemap:index.xml.twig'), $this->anything())
+            ->will($this->returnValue('some-xml-string'));
 
         /** @var Response $response */
         $response = $this->controller->indexAction('xml', 'test');
 
-        $this->assertEquals(new Response('some-xml-string'), $response->getContent());
+        $this->assertEquals('some-xml-string', $response->getContent());
```

Comment:
this was the actual cause of the random test failures: We compared a Response object to `$response->getContent()`. That means that PHP/HHVM needed to convert the first argument to a string before comparing with getContent. Some versions of HHVM did not do this and threw an exception: Comparing an object with a string